### PR TITLE
printing packet information

### PR DESCRIPTION
**Pull request by wkerzendorf**
when the MC crashes with diff nu < 0

### CODE REVIEW

File: tardis/montecarlo/montecarlo_numba/r_packet.py [hunk 1]
```diff
@@ -68,3 +68,12 @@ def initialize_line_id(self, numba_plasma, numba_model):
         self.next_line_id = next_line_id
 
 
+@njit(**njit_dict_no_paralel)
+def print_r_packet_properties(r_packet):
```
Comment (andrewfullard) [inline tardis/montecarlo/montecarlo_numba/r_packet.py:72]: docstring

Comment (wkerzendorf) [reply tardis/montecarlo/montecarlo_numba/r_packet.py:72]: 😝

**Review by andrewfullard — APPROVED** [review]

**Review by andrewfullard — CHANGES_REQUESTED** [review]
njit missing from import

File: tardis/montecarlo/montecarlo_numba/r_packet.py [hunk 2]
```diff
@@ -68,3 +68,18 @@ def initialize_line_id(self, numba_plasma, numba_model):
...
+    Parameters
+    ----------
+    r_packet : RPacket
+        RPacket object
+    """
+    print("-"*80)
+    print("R-Packet information:")
+    for r_packet_attribute_name, _ in rpacket_spec:
+        print(r_packet_attribute_name, "=", str(getattr(r_packet, r_packet_attribute_name)))
+    print("-"*80)
```
Comment (afloers) [inline tardis/montecarlo/montecarlo_numba/r_packet.py:85]:
```suggestion
    print("-"*80)
    
```

**Review by andrewfullard — APPROVED** [review]

**Pull request merged**